project_definition/README.rst

```diff
@@ -0,0 +1,54 @@
+. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============================
+Term definitions in projects
+============================
+
+Adds a new document page type "Term" to create dictionary-style word archive and
+definition. 
+Adds Searchable view for terms and definitions in the Knowledge menu.
+Has a search view per term and per definition and a dictionary Style Alphabetical grouping view.
+Shows term definitions on every project. just adds a relation between
+document_definition  and project models and adds a tab to the project view.
```

Comment:
Can we have a usage example, showcasing a situation where this feature is needed?

Reply:
In out local usage we needed a different place to have terms and definitions to attach to a project.  We wanted to create a "terms and definitions" type with it's own menu, search views and  filters without fractioning the KB with a new model.
So we extended with a new type "term"
THe advantage of this is that such "terms" are visible in their own section, and when attaching terms to a project our selection will show only available terms , not all pages of the KB.